docs/blog/newsletter-december-2018.rst

```diff
@@ -0,0 +1,128 @@
+
+- Swapnil's `blog post roundup <TODO>`_ 
+- A lovely `playlist of all the videos <https://www.youtube.com/watch?v=N13_FP2NkSk&list=PLy70RNJ7dYrJ1wANiqa7ObwUnoJjouQjt>`_ 
+- The fabulous `photo album <https://www.flickr.com/photos/writethedocs/albums/72157697741784910>`_
+
+Looking forward to next year: the Call for Proposals for Write the Docs Portland has just gone up! It's open until **midnight PST, 17 January 2019**, so get cracking and send us all your brilliant talk ideas. For full details, here's the `announcement post <http://www.writethedocs.org/conf/portland/2019/news/welcome/>`_. 
+
+The newsletter team is off next month for Christmas (it's snowing here already and I am *super* excited); we'll be back in your inboxes in February. Happy holidays, everyone!
+
+----------------------
+Discovering user needs
+----------------------
+
+Understanding what users need is at the heart of good technical writing. A recent question about how to determine user needs led to tons of ideas about who to ask, how to ask, and what to watch out for:
+
+* Ask your colleagues: product managers, support/success teams, developers...
+
+  Product mangers often have wide knowledge about user needs. Support can tell you about big and small customer pain points. Developers - particularly experienced ones - can give you insight into use cases, and the needs their code seeks to address.
+* Also try reviewing support tickets, for a detailed view on how users really work with the product.
+* Take a look at customer requests, and analyse them carefully. You want to distinguish between what customers ask for, and what they actually need.
+
+  If you step back, you may realise that behind a specific request, there's a better solution to their problem -- teleporting as opposed to a faster car was one humorous example.
+* Dig through StackOverflow for questions tagged with your product (or related keywords).
+* Communicate with users to let them know that you're working on their issues. Even if it's not strictly speaking a docs issue, your communication skills can help product management and support reassure customers who are waiting for new features.
+
+------------------------------
+Technical writing career paths
+------------------------------
+
+There was a big discussion this month about how to define career paths for tech writers. Even the term itself was controversial, spanning two related ideas:
+
+- How a company defines levels of a role (also called  a "career ladder").
+
+  This could be levels like Associate -> Senior -> Principal/Lead, or Technical Writer Level 1 through 4 (or higher).
+
+- Where you personally go in your career.
+
+  This isn't bounded by your organisation's ladder: your path might involve changing company, or role. You might have a five-year plan, or think one step at a time, or no plan at all.
+
```

Comment:
Feels to me like there is quite a bit of repetition of ladder / collab / no set path, ... if you're looking for somewhere to squish: like 48-52 AND 56 all reads like one para?

Reply:
Good spot!